Automattic/woocommerce-services
# Implement npm shrinkwrapping

This PR seeks to shrinkwrap the dependencies (see https://docs.npmjs.com/cli/shrinkwrap) -- fixes 98

It also moves wp-calypso to a regular dependency.

To test:
- Checkout this branch
- `rm -rf node_modules`
- `npm install` -- spot check that dependencies are installed correctly
- `npm test` -- make sure that all tests pass (don't worry about the `propType` warning)
- `npm run webpack` to genreate the js
- `./node_modules/.bin/node-sass assets/stylesheets/style.scss --include-path './node_modules/wp-calypso/client:./node_modules/wp-calypso/assets/stylesheets' dist/style.css` to generate the stylesheet
- verify that the shipping settings form still renders (as per screenshot below)

![screenshot from 2016-03-23 11-22-41](https://cloud.githubusercontent.com/assets/260253/13996126/9a5cd242-f0e9-11e5-85ec-a8d1202c3102.png)

**IMPORTANT**: from now on, if you install or update an npm package, you'll have to run `npm shrinkwrap` too

cc @nabsul @allendav @jeffstieler
